src/dmd/frontend.h

```diff
@@ -1150,6 +1150,7 @@ struct FileName
     static const char* canonicalName(const char* name);
     static void free(const char* str);
     const char* toChars() const;
+    template <typename T>
```

Comment:
Still searching for this bug, apparently `linkage` is not reset correctly. 
But `Filename`  shouldn't even be emitted as it's `extern(D)` ...

Reply:
Implemented a way to detect if `onemember` was actually emitted.